src/main/java/de/unijena/cheminf/mortar/controller/HistogramViewController.java

```diff
@@ -964,10 +964,9 @@ private StackPane createStackPaneWithContextMenuAndStructureDisplayForBar(ImageV
                 tmpContextMenuLabel.setMinWidth(tmpNodePane.getWidth());
                 tmpContextMenuLabel.setTranslateX(0);
                 //TODO: @Betül, why do you use different listeners here and below? And why the context menu label color setting in only one of them?
-                tmpContextMenuLabel.addEventHandler(MouseEvent.MOUSE_CLICKED,event1 -> {
-                    if (MouseButton.SECONDARY.equals(event1.getButton())) {
-                        tmpContextMenuLabel.setStyle("-fx-bar-fill: " + HistogramViewController.HISTOGRAM_BARS_SELECTED_COLOR_HEX_VALUE);
```

Comment:
Sorry for doubling this, but why did you deleted this line?

Reply:
See a comment done by Betül and the discussion about it above. According to her, this line was actually not needed. I think she is going to work on this part herself now as well, because there is some confusion about this block.